mlrun/api/db/sqldb/db.py

```diff
@@ -490,6 +490,12 @@ def create_schedule(
             cron_trigger=cron_trigger,
         )
 
+        try:
+            labels = get_in(scheduled_object, "metadata.labels", {})
+        except TypeError:
+            labels = {}
+        update_labels(schedule, labels)
+
```

Comment:
Much more then this is needed to add support for Schedule labels:
1. you need to delete the labels when deleting the schedule
2. you need to add the labels on the schedule resource on get
3. you need to make changes in schemas (not DB models, schemas, for the API) to support returning the labels on get

I'm thinking maybe better in this PR to just add support in adding the table + relationship to the DB, and add the real support in a followup PR

Reply:
Indeed. The only reason I added “support” was because tests were failing without this support. And the PR was broken. 
What do you suggest if not adding these things here?